### PR TITLE
Kolibri Design System - Buttons

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary

~**WIP**~

1. Implement KButtonGroup (done) 
2. Implement KIconButton in lieu of UiIconButton
3. Misc KDS related fixes (ie, wrongly named icon that I found).

### Reviewer guidance

**General information about how to test this**

See: https://github.com/learningequality/kolibri-design-system/pull/16

**KButtonGroup**

Here is where I added the KButtonGroup - check these out:

- Facility Settings buttons
- Facility > Data - "Sync facility data" buttons right of the name of your facility.
- Device > Channels > Manage Channel
- Device > Channels > Import by network modal
- Facility > Users > New user form
- Facility > Users > Edit user form
- Learn > Quiz bottom bar
- Edit profile page
- I added this in a few other places since writing this description originally.

Here are a few other notes:

- Verify that I've not missed any places where two KButtons are near each other.
- Also verify that other button placements are not out of wack. It is worth noting that top-right table buttons (Coach / Facility) are now aligned flush with the right side of their container. There are likely other places where this may be noticeable.

### References

https://github.com/learningequality/kolibri/issues/5900

Kolibri Design System PR: https://github.com/learningequality/kolibri-design-system/pull/16

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by indirectlylit** [comment]
Looking good!

See note on button alignment here: https://github.com/learningequality/kolibri-design-system/pull/16#issuecomment-604297500

**Comment by indirectlylit** [comment]
@nucleogenesis any idea what the build failure is from?

https://travis-ci.org/github/learningequality/kolibri/jobs/670347824?utm_medium=notification&utm_source=github_status

File: kolibri/plugins/coach/assets/src/views/common/BackLink.vue [hunk 1]
```diff
@@ -2,33 +2,16 @@
...
-      aria-hidden="true"
-      @click="go"
-    >
-      <KIcon
-        icon="back"
-        :style="{ fill: $themeTokens.primary, top: 0, height: '24px', width: '24px' }"
-      />
-    </UiIconButton>
-    <KRouterLink :to="to" :text="text" />
+    <KRouterLink :to="to" icon="back" :text="text" />
```
Comment (indirectlylit) [inline kolibri/plugins/coach/assets/src/views/common/BackLink.vue:5]: Looks like we might be able to delete the `BackLink` component entirely and replace every instance with just

```html
<KRouterLink :to="target" icon="back" :text="text" />
```

**Review by indirectlylit — APPROVED** [review]
this is great! tested and it's working well.

merging and we can address new things in follow-up issues

**Comment by nucleogenesis** [comment]
> @nucleogenesis any idea what the build failure is from?
> 
> https://travis-ci.org/github/learningequality/kolibri/jobs/670347824?utm_medium=notification&utm_source=github_status

No idea - I'll give it a look later on.

**Pull request merged**